Scoop/Model/Request.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Request.swift
+//  Scoop
+//
+//  Created by Tiffany Pan on 4/4/23.
+//
+
+import Foundation
+
+struct Request: Codable {
```

Comment:
Hmm. `Request` seems very vague. Perhaps maybe renaming it to `RideRequest`?

Reply:
good point, thx vin!